docs/resources/mlflow_model_serving.md

```diff
@@ -0,0 +1,88 @@
+---
+subcategory: "MLflow"
+---
+# databricks_mlflow_model_serving Resource
+
+This resource allows you to manage [Model Serving Endpoints](https://docs.databricks.com/machine-learning/model-inference/serverless/serverless-real-time-inference.html) in Databricks.
```

Comment:
the link will change - will let you know when it's final

Reply:
@sueann any update on the link?

Reply:
I see the link auto-redirects to https://docs.databricks.com/machine-learning/model-serving/index.html so gonna use that